pokemongo_bot/walkers/walker_factory.py

```diff
@@ -8,5 +8,8 @@ def walker_factory(name, bot, dest_lat, dest_lng, dest_alt=None, *args, **kwargs
     if 'StepWalker' == name:
         ret = StepWalker(bot, dest_lat, dest_lng, dest_alt)
     elif 'PolylineWalker' == name:
-        ret = PolylineWalker(bot, dest_lat, dest_lng)
+        mode = "walking"
+        if "mode" in kwargs:
+            mode = kwargs["mode"]
+        ret = PolylineWalker(bot, dest_lat, dest_lng, mode=mode)
```

Comment:
```
    elif 'PolylineWalker' == name:
        ret = PolylineWalker(bot, dest_lat, dest_lng, mode=kwargs.get('mode', walking'))
```

instead of:

```
    elif 'PolylineWalker' == name:
        mode = "walking"
        if "mode" in kwargs:
            mode = kwargs["mode"]
        ret = PolylineWalker(bot, dest_lat, dest_lng, mode=mode)
```